docker_enforcer.py

```diff
@@ -193,8 +194,10 @@ def authz_request():
     verdict = requests_judge.should_be_killed(json_data)
     if verdict.verdict:
         return process_positive_verdict(verdict, json_data, register=False)
+
+    containers_regex = re.compile("^(/v.+?)?/containers/.+?$")
```

Comment:
can you move the `re.compile` to initialization code, to avoid compiling it with each method call?